src/Components/People.js

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+
+export class People extends Component {
+    constructor(){
+        super()
+
+        this.state = {
+            input: "",
+            people: [],
+            name: "",
+            age: "",
+            gender: "",
+            isError: false,
+        }
+    }
+
+    componentDidMount () {
+        this.getPeople()
+    }
+
+    getPeople = async () => {
+        const {data} = await axios.get(`https://ghibliapi.herokuapp.com/people`)
+        // console.log(data)
+
+        this.setState ({
+            people: data,
+        })
+    }
+
+    handleInput = (event) => {
+        this.setState({
+            input: event.target.value,
+        })
+    }
+
+    handleSubmit = async (event) => {
+        event.preventDefault()
+
+        const {input,people} = this.state
+        const person = people.filter((p) => p.name === input)
```

Comment:
since you will only show at most one person, .find() might be a more appropriate method to use than .filter()